src/Service.Tests/SqlTests/RestApiTests/RestBodyFlexibleModeTests/RestBodyNonStrictModeTests.cs

```diff
@@ -0,0 +1,196 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Net;
+using System.Threading.Tasks;
+using Azure.DataApiBuilder.Config.ObjectModel;
+using Azure.DataApiBuilder.Service.Exceptions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Azure.DataApiBuilder.Service.Tests.SqlTests.RestApiTests
+{
+    /// <summary>
+    /// Class containing integration tests to validate scenarios when we operate in non-strict mode for REST request body,
+    /// i.e. we allow extraneous fields to be present in the request body.
+    /// </summary>
+    public abstract class RestBodyNonStrictModeTests : RestApiTestBase
+    {
+        #region Positive tests
+
+        /// <summary>
+        /// Test to validate that extraneous fields are allowed in request body when we operate in runtime.rest.request-body-strict = false.
+        /// When PK fields are specified both in URI and in the request body, precedence is given to the values specified for the fields in the URI.
```

Comment:
After looking at RFC again.. instead of `precedence is given`, you should say something like 'only the values specified for the fields in the URI are honored.' and values specified for fields in the body are ignored